dataland-frontend/src/components/forms/parts/elements/derived/ActivitySelector.vue

```diff
@@ -43,13 +43,14 @@
       ref="multiSelectFormFieldRef"
       dataTest="selectNaceCodes"
       name="naceCodes"
-      :emptyMessage="`No NACE code available for ${selectedActivities.name} Activity`"
-      :validation="naceCodesForActivities.length ? 'required' : ''"
```

Comment:
Removing this line fixes the problem, and the remaining changes are typescript related?

Reply:
Yes. Zeile 47 hier ist der eigentliche change. Alles andere is fix-up